integration_tests/src/main/python/json_test.py

```diff
@@ -1396,7 +1395,6 @@ def test_spark_from_json_empty_table(data):
         conf =_enable_all_types_conf)
 
 # SPARK-20549: from_json bad UTF-8
-@pytest.mark.xfail(reason='https://github.com/NVIDIA/spark-rapids/issues/10483')
```

Comment:
I am a little confused here. https://github.com/NVIDIA/spark-rapids/issues/10483 is closed. I am guessing that I messed enabling this test again some how, but I thought I had gone through all of them. Oh well. So what is the issue that is blocking this from working properly? Or is it just that I forgot to mark it as working?

Reply:
I verified with the latest code that the issue #10483 is fixed thus I think this is just forgotten in your last PR.